src/components/Conclusion/index.jsx

```diff
@@ -112,6 +112,10 @@ const weightTableContent = [
 ]
 
 function averageDuration(data) {
+  if (typeof (data) == "undefined") {
+    return 0
+  }
+
```

Comment:
it's unrecommended to use "==", because it leads to unexpected errors
in this case, you can write like 
`if(data) 0`